spanner/spanner_snippets/snippet.go

```diff
@@ -906,6 +916,264 @@ func queryWithTimestamp(ctx context.Context, w io.Writer, client *spanner.Client
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			rowCount, err := txn.Update(ctx, spanner.Statement{
+				SQL: `INSERT Singers (SingerId, FirstName, LastName)
+										VALUES (10, 'Virginia', 'Watson')`})
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(w, "%d record(s) inserted.\n", rowCount)
+			return nil
+		})
+	return err
+}
+
+// [END spanner_dml_standard_insert]
+
+// [START spanner_dml_standard_update]
+
+func updateUsingDML(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			rowCount, err := txn.Update(ctx, spanner.Statement{
+				SQL: `UPDATE Albums
+								SET MarketingBudget = MarketingBudget * 2
+								WHERE SingerId = 1 and AlbumId = 1`})
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(w, "%d record(s) updated.\n", rowCount)
+			return nil
+		})
+	return err
+}
+
+// [END spanner_dml_standard_update]
+
+// [START spanner_dml_standard_delete]
+
+func deleteUsingDML(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			rowCount, err := txn.Update(ctx, spanner.Statement{
+				SQL: `DELETE Singers WHERE FirstName = 'Alice'`})
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(w, "%d record(s) deleted.\n", rowCount)
+			return nil
+		})
+	return err
+}
+
+// [END spanner_dml_standard_delete]
+
+// [START spanner_dml_standard_update_with_timestamp]
+
+func updateUsingDMLWithTimestamp(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			rowCount, err := txn.Update(ctx, spanner.Statement{
+				SQL: `UPDATE Albums
+								SET LastUpdateTime = PENDING_COMMIT_TIMESTAMP()
+								WHERE SingerId = 1`})
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(w, "%d record(s) updated.\n", rowCount)
+			return nil
+		})
+	return err
+}
+
+// [END spanner_dml_standard_update_with_timestamp]
+
+// [START spanner_dml_write_then_read]
+
+func writeAndReadUsingDML(ctx context.Context, w io.Writer, client *spanner.Client) error {
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+			// Insert Record
+			rowCount, err := txn.Update(ctx, spanner.Statement{
+				SQL: `INSERT Singers (SingerId, FirstName, LastName)
+								VALUES (11, 'Timothy', 'Campbell')`})
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(w, "%d record(s) inserted.\n", rowCount)
+
+			// Read newly inserted record
+			iter := txn.Query(ctx, spanner.Statement{
+				SQL: `SELECT FirstName, LastName FROM Singers WHERE SingerId = 11`})
```

Comment:
Either move the `}` to a newline or move the `spanner.Statement` declaration to its own line (and create a statement variable).